### DISCUSSION TITLE
reComputer R1000 + Raspberry Pi AI Kit

### DISCUSSION THREAD

**Discussion by jennazhuxj** [Seeed-Studio/OSHW-RPi-Series#10] [On Development]
Do you need a reComputer R1000 integrated with the latest AI Kit just announced by Raspberry Pi for your application?
How do you plan to deploy it and for which scenarios? Let us know and we help make it happen!
![ai kit](https://github.com/Seeed-Studio/reComputer_R_Series/assets/113671654/f052b357-54bc-490b-802f-842fbd10b803)

**Comment by LJ-Hao** [comment]
Yes, I performed YOLOv8 object detection benchmarking with the R1000 and AI Kit and found that the FPS for object detection can reach 29.5. I believe using this in the security and retail sectors would be an excellent choice.

**Comment by jennazhuxj** [comment]
Check out our upcoming new live show where we will have a pre-showcase about the new device integrated CM4 and AI kit.
https://www.youtube.com/watch?v=ItbletB9N7I
